### PR TITLE
Flaggedmsgs

### PR DESCRIPTION
**Pull request by hussain-mohammed**
- Added is_flagged in the messages model
- Updated the Flagged messages API integration
- Displaying bot messages on the right side
- Added username with the message in individual chat

@kaapstorm

### CODE REVIEW

**Comment by hussain-mohammed** [comment]
Added websockets in the UI for real-time messages and unread messages count. 

@kaapstorm

File: static/src/views/Chat.vue [hunk 1]
```diff
@@ -48,7 +48,7 @@ export default {
       toastMessage: '',
       showToastError: false,
       rooms: [],
-      messages: [],
+      // messages: [],
```
Comment (kaapstorm) [inline static/src/views/Chat.vue]: Fine just to delete

```suggestion
```

File: static/src/services/MessageService.js [hunk 2]
```diff
@@ -12,7 +13,7 @@ export default {
         message: content,
         from_group: groupView,
       });
-      return response;
+      // return response;
```
Comment (kaapstorm) [inline static/src/services/MessageService.js]: Fine just to delete

```suggestion
```

**Review by kaapstorm — APPROVED** [review]
:+1:

**Pull request merged**